pkg/config/apiserver.go

```diff
@@ -206,12 +252,10 @@ func (a *API) getScopes(req *restful.Request, resp *restful.Response) {
 	writeErrorResponse(http.StatusNotImplemented, "Listing scopes not implemented", resp)
 }
 
-// getRules returns the rules document for the scope and the subject.
+// getConfig returns the rules document for the scope and the subject.
```

Comment:
These are API methods. They `return` from the http caller perspective by writing to the `resp` struct.